### PR TITLE
Fix compiler warnings rather than suppress them

### PR DESCRIPTION
**Pull request by lzaoral**
See the commits.

### CODE REVIEW

File: analyses/dg_points_to_plugin.cpp [hunk 1]
```diff
@@ -436,7 +436,7 @@ std::string PointsToPlugin::pointsTo(llvm::Value* a, llvm::Value* b) {
 std::string PointsToPlugin::pointsToSetsOverlap(llvm::Value* a, llvm::Value* b) {
     assert(PTA);
     PSNode *nodeA = PTA->getPointsToNode(a);
-    PSNode *nodeB = PTA->getPointsToNode(a);
+    PSNode *nodeB = PTA->getPointsToNode(b);
```
Comment (mchalupa) [inline analyses/dg_points_to_plugin.cpp:439]: Could you put this change into a separate commit? It changes the behavior of the method. Thanks for finding it!

**Review by mchalupa — CHANGES_REQUESTED** [review]
I really like this PR, thanks. I have just one comment regarding the structure of commits.

File: analyses/dg_points_to_plugin.cpp (hunk 1 above)
Comment (lzaoral) [reply analyses/dg_points_to_plugin.cpp:439]: Done!

**Comment by mchalupa** [comment]
Great! Thank you. Merged.
